recipes-devtools/nsight-systems/nsight-systems-cli_2020.5.3.17-1.bb

```diff
@@ -0,0 +1,53 @@
+SUMMARY = "NVIDIA Nsight Systems is a multi-core CPU sampling profiler that \
+    provides an interactive view of captured profiling data, helping improve \
+    overall application performance."
+HOMEPAGE = "https://developer.nvidia.com/nsight-systems"
+LICENSE = "Proprietary"
+LIC_FILES_CHKSUM = " \
+    file://opt/nvidia/nsight-systems-cli/2020.5.3/NVIDIA_SLA.pdf;md5=f7646eea4f29c4f708ea0aa5bc6778a6 \
+"
+
+inherit nvidia_devnet_downloads
+
+SRC_URI = "${NVIDIA_DEVNET_MIRROR}/nsight-systems-cli-2020.5.3_${PV}_arm64.deb;subdir=nsight-systems-cli"
```

Comment:
It looks like this package is available off NV's debian package feed.  The name is a bit different, but the sha256sum is identical.  So you could instead use
```
inherit l4t_deb_pkgfeed

SRC_COMMON_DEBS = "${BPN}-2020.5.3_2020.5.3.17-0256620_arm64.deb;subdir=${BPN}"
```
which would make this buildable without having the package in NVIDIA_DEVNET_MIRROR.

Reply:
Perfect, I tried two days ago with different package name but I didn't successfully download the package.
Thanks.